python/python/lancedb/rerankers/base.py

```diff
@@ -130,12 +134,91 @@ def merge_results(self, vector_results: pa.Table, fts_results: pa.Table):
         combined = pa.concat_tables(
             [vector_results, fts_results], **self._concat_tables_args
         )
-        row_id = combined.column("_rowid")
 
         # deduplicate
-        mask = np.full((combined.shape[0]), False)
+        combined = self._deduplicate(combined)
+
+        return combined
+
+    def rerank_multivector(
+        self,
+        vector_results: Union[List[pa.Table], List["LanceVectorQueryBuilder"]],
+        query: Union[str, None] = None,  # Some rerankers might not need the query
```

Comment:
Here, I was not sure about order of the args. `query` comes 2nd compared to other rerank_ methods because:
- These are meant to be directly called by the user 
- And `query` might be optional for some rerankers

Reply:
I think the order is fine.  However, maybe `query` should not have a default value but clearly document that it could be None?  This way you force the user to explicitly state "I know my reranker does not need the query"?

Reply:
yeah sounds good.. Updated